Readme_TimeInCapitals.md

```diff
@@ -0,0 +1,25 @@
+# TimeInCapitals
+
+>Simple API that allows you to get time for capitals
+
+## Table of contents
+* [General info] (#general-info)
+* [Technologies] (#technologies)
+* [Setup] (#setup)
+* [Features] (#features)
+
+## General info
+TimeInCapitals is a SpringBoot app.
+
+Simple API that displays the time in a specific capital.
+
+## Technologies
+* SpringBoot 2.1.7
+* JAVA 1.8
```

Comment:
Java

Reply:
Poprawiono

Reply:
Chodziło mi o to, żeby nie pisać JAVA DUŻYMI literami, a nie wywalić wszystko

Reply:
Poprawione

Reply:
Nie widzę zmiany?

Reply:
Sorka, coś musiałem pomylić, ale teraz już poprawiłem, przywróciłem też SprinBoota, czy tak jak jest teraz jest OK?